dirty_equals/_boolean.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, List, Optional, Set, Tuple, Type, TypeVar, Union, _SpecialForm
+
+from ._base import DirtyEquals
+
+B = TypeVar('B', int, bool, str, float, List[Any], Dict[Any, Any], Set[Any], Tuple[Any], Type[None])
+
+
+class IsFalseLike(DirtyEquals[B]):
+    """
+    Check if the value is False like, and matches the given conditions.
+    """
+
+    allowed_types: Union[Type[B], Tuple[Union[_SpecialForm, type], ...]] = (
```

Comment:
There might be a much simpler way to do this, I got quite confused working with `_SpecialForm` and mypy here. This also requires a type ignore [here](https://github.com/osintalex/dirty-equals/blob/e14b834bab5720ba3b9646c9a26a832e41f68175/dirty_equals/_boolean.py#L60).